### PR TITLE
8.x drush export

### PR DESCRIPTION
**Pull request by dawehner**
Partly solves https://github.com/larowlan/default_content/issues/10

### CODE REVIEW

**Comment by jibran** [comment]
Very nice idea. We should have unittest for DefaultContentManager maybe follow up.

File: src/DefaultContentManagerInterface.php [hunk 1]
```diff
@@ -31,4 +31,17 @@ public function setScanner(DefaultContentScanner $scanner);
...
+  /**
+   * Exports a single entity as importContent expects it.
+   *
+   * @param string $entity_type_id
+   *   The entity type ID.
+   * @param $entity_id
+   *   The entity ID to export.
+   *
+   * @return string
+   *   The rendered export as hal.
```
Comment (andypost) [inline src/DefaultContentManagerInterface.php:43]: hal+json? as I know it could be hal+xml

Comment (dawehner) [inline src/DefaultContentManagerInterface.php:43]: https://github.com/larowlan/default_content/blob/8.x/src/DefaultContentManager.php#L146 uses hal_json on the other side of the table

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "drupal/default_content",
+  "description": "Imports default content when a module is enabled",
+  "type": "drupal-module",
+  "license": "GPL-2.0+",
+  "minimum-stability": "dev",
+  "require": { }
```
Comment (larowlan) [inline composer.json:7]: Should this reference drupal/core?

**Pull request merged**